howso/details_residuals.amlg

```diff
@@ -56,29 +72,51 @@
 		))
 
 		;append feature deviations to output if selected
-		(if (get details "feature_residuals")
-			(accum (assoc output (assoc "feature_residuals" (get local_feature_metrics "residual_map"))))
+		(if (or (get details "feature_residuals_full") (get details "feature_residuals_robust"))
+			(if robust_residuals
+				(accum (assoc output (assoc "feature_residuals_robust" (get local_feature_metrics "residual_map"))))
+				;else full residuals
+				(accum (assoc output (assoc "feature_residuals_full" (get local_feature_metrics "residual_map"))))
+			)
 		)
 
 		;append prediction stats to output if selected
-		(if (get details "prediction_stats")
-			(accum (assoc 
-				output 
-					(assoc 
-						"prediction_stats" 
-							(append 
-								(remove 
-									(get local_feature_metrics "prediction_stats")
-									;Keep the confusion matrices if it is selected for
-									(if (get details "confusion_matrices")
-										(list ".hyperparam_path" ".robust")
-										(list ".hyperparam_path" ".robust" "confusion_matrix")
+		(if (or (get details "prediction_stats") (get details "prediction_stats_robust"))
+			(if robust_residuals
+				(accum (assoc
```

Comment:
Similarly to another comment I left, you have two branches of an if that both `(accum)` to the same variable. In these cases, the assignment/accumulating opcode can be on the outer layer and the if/branches can be moved inward. 

A few less opcodes/nodes this way and more functional :)